exercises/saddle-points/src/main/java/Matrix.java

```diff
@@ -0,0 +1,13 @@
+import java.util.List;
+import java.util.Set;
+
+class Matrix {
+
+    Matrix(List<List<Integer>> lists) {
```

Comment:
Maybe calling this parameter something like `values` or `matrixValues` might make it a bit more clear what it is? :)

Reply:
Sure thing!